sap-schemas/v2.0/README.md

```diff
@@ -368,3 +368,31 @@ Example: `DECIMAL(23,2)` value for `price`
   }
 }
 ```
+
+### `x-sap-root-entity`
+
+- Type: `Boolean`
+- Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
+- Description: Indicates if the marked object represents a root entity.
```

Comment:
As the concept of a root entity is not available in OpenAPI, we should have a few explanatory words.

Reply:
Please make a suggestion for the description extension.